src/util/apiHelpers.js

```diff
@@ -49,7 +49,7 @@ export const fetchWithAuthorization = async (url, config = {}, forceAuth) => {
     ? config.headers['Content-Type']
     : 'text/plain';
   const extraHeaders = contentType ? { 'Content-Type': contentType } : {};
-  const cacheControl = ['POST', 'PUT', 'PATCH', 'DELETE'].includes(
+  const cacheControl = ['GET', 'POST', 'PUT', 'PATCH', 'DELETE'].includes(
```

Comment:
Hmm, var vel en grunn til at vi ikke inkluderte GET her. Hvorfor endret du det?

Reply:
Syns det er uheldig å returnere cachede resultater når vi skal redigere på dem.

Kall til taksonomi caches, så om du endrer ressurstypen to ganger før cachen går ut (10min) så vil du forsøke å slette de gamle connectionene (som allerede er slettet), og de som du faktisk vil slette vil bli liggende.

Hvis vi har lyst til å cache GET requester så burde vi invalidere den cachingen av den url'en hver gang vi oppdaterer taksonomi, og det høres ikke gøy ut.

Tipper vi har så få brukere at de ikke engang vil merke forskjellen.